src/actions/lookup.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @module "actions.lookup"
+ * @desc Actions related to lookup data.
+ */
```

Comment:
Yes we need lookup module.

Such lookup data can be read/referenced by all members, instead of restricted to some specific member. Typically they are used as dropdown options for member to choose.

The skill tags is exactly such kind of lookup data.

Reply:
@liuliquan well, the intention of `member` modules is not to handle only stuff related to a single member, but to handle anything related to members api functionality.

Reply:
But the member/profile action certainly looks like that way, all the functions have a "handle" parameter for a specific member.

And the skill tags is only one kind of tag. There could be other kind of tags to be added in future.

This would be the first function added that does not take a "handle" parameter, are you sure for this?

Reply:
Yes, if I get it right, that the purpose of that thing is to get skills of the authenticated user from the API. If it belongs by purpose to member-related functionality, let's blend it into `member`  or `profile` module. If we create a separate module for each new function just because it has slightly different signature, we end up with too much boilerplate code, and it will become always hard to find whether some functionality already exists, and where it is.

Reply:
ok, nevermind, let's keep it as it for now, and then figure out where to move it later.